engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java

```diff
@@ -168,14 +168,14 @@ public ValidatorFactoryImpl(ConfigurationState configurationState) {
 			tmpFailFast = hibernateSpecificConfig.getFailFast();
 
 			tmpAllowOverridingMethodAlterParameterConstraint =
-					hibernateSpecificConfig.getMethodValidationConfiguration()
-							.isAllowOverridingMethodAlterParameterConstraint();
+							hibernateSpecificConfig.getMethodValidationConfiguration()
+											.isAllowOverridingMethodAlterParameterConstraint();
```

Comment:
Formatting issue. It would be nice if you could review your patch with Eclipse or something and revert all the formatting noise.